### PR TITLE
feat: AKS-Engine Install Script for Linux 

### PR DESCRIPTION
**Pull request by palma21**
**Reason for Change**:

Providing a simple install script to automatically pull and install into the Path the desired AKS Engine version. Defaults to latest
Also upgrades existing versions.

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

### CODE REVIEW

File: scripts/get_akse.sh [hunk 1]
```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
+
+# Copyright Jorge Palma.
```
Comment (mboersma) [inline scripts/get_akse.sh]: We're supposed to have the Microsoft standard copyright header on each code file, which I think would include scripts as well, although we haven't been doing that. I'm not a lawyer, but I think asserting a different copyright might be problematic.

Comment (palma21) [reply scripts/get_akse.sh]: Yup, copy pasta foo. I technically think it should be alright but def not gonna dive into that. Changed.

**Comment by mboersma** [comment]
lgtm

**Pull request merged**